packages/fast-components-react-base/app/examples.ts

```diff
@@ -1,2 +1,4 @@
 import ButtonExamples from "../src/button/examples.data";
-export {ButtonExamples};
+import MediaExamples from "../src/media/examples.data";
+
+export {ButtonExamples, MediaExamples};
```

Comment:
I think we'll want to keep these separate. This single export is going to end up spanning a multitude of lines as we add components.